src/xbot/common/math/PIDPropertyManager.java

```diff
@@ -129,4 +146,12 @@ public boolean getEnableDerivativeThreshold() {
     public void setEnableDerivativeThreshold(boolean isEnabled) {
         propEnableDerivativeThreshold.set(isEnabled);
     }
+    
+    public boolean getEnableTimeThreshold() {
+        return propEnableTimeThreshold.get();
```

Comment:
I was just wondering if this was a thing we really needed enable/disable around or if a threshold of 0 is pretty self explanatory in this case?